src/model/Model.cpp

```diff
@@ -21,8 +21,11 @@
 #include <utilities/idd/IddEnums.hxx>
 #include <utilities/idd/OS_Version_FieldEnums.hxx>
 
+#include "../osversion/VersionTranslator.hpp"
```

Comment:
So, there's one thing I'm not sure about here. This ends up referencing the osversion::VersionTranslator, when the osversion::VersionTranslator itself references the model/Schedule.hpp (for some 0.8.x translation). osversion depends on model at the model. But all of them are OBJECT libraries anyways, so that looks to link just fine in the end... @kbenne thoughts please?